### ISSUE TITLE
parse_title no longer works with the most recent version of the tidyverse.

### ISSUE THREAD

**Issue by mattfidler** [UUPharmacometrics/xpose#105]
Hi @guiastrennec ,

With the most recent tidyverse/ggplot2 (v 3) the `parse_title` no longer works as expected.   Here is the theo example using xpose.nlmixr theophille data.

```R
library(nlmixr)
library(xpose)
library(xpose.nlmixr)

theoSd <- function() {
    ini({
        tka <- .5
        tcl <- -3.2
        tv <- -1
        eta.ka ~ 1
        eta.cl ~ 2
        eta.v ~ 1
        add.err <- 0.1
    })
    model({
        ka <- exp(tka + eta.ka)
        cl <- exp(tcl + eta.cl)
        v <- exp(tv + eta.v)
        linCmt() ~ add(add.err)
    })
}

theoFit <- nlmixr(theoSd, theo_sd, est="saem")

xpdb <- xpose_data_nlmixr(theoFit)
dv_vs_pred(xpdb)
```

When you run dv_vs_pred you get:
```R
> dv_vs_pred(xpdb)
Using data from $prob no.1
Filtering data by EVID == 0
Warning message:
y, x is not part of the available keywords. Check ?template_titles for a full list. 
> 
```

![image](https://user-images.githubusercontent.com/514778/42700469-73699214-8689-11e8-8a73-ae260017c4c5.png)

I'm not sure which of the tidyverse packages is broken.  But you could do the same thing using Base R.  However tidyverse fans don't often want to do that, so I will let you decide.

I'm copying @kestrel99  for awareness.

Here is my sessionInfo:
<details>

```R
> sessionInfo()
R version 3.5.1 (2018-07-02)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 10 x64 (build 15063)

Matrix products: default

locale:
[1] LC_COLLATE=English_United States.1252 
[2] LC_CTYPE=English_United States.1252   
[3] LC_MONETARY=English_United States.1252
[4] LC_NUMERIC=C                          
[5] LC_TIME=English_United States.1252    

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] bindrcpp_0.2.2     xpose.nlmixr_0.1.2 xpose_0.4.3        ggplot2_3.0.0     
[5] nlmixr_0.9.1-3    

loaded via a namespace (and not attached):
 [1] tidyselect_0.2.4          purrr_0.2.5              
 [3] lattice_0.20-35           colorspace_1.3-2         
 [5] RxODE_0.7.2-5             vpc_1.0.2                
 [7] rlang_0.2.1               R.oo_1.22.0              
 [9] pillar_1.2.3              glue_1.2.0               
[11] withr_2.1.2               R.utils_2.6.0            
[13] tweenr_0.1.5              bindr_0.1.1              
[15] plyr_1.8.4                stringr_1.3.1            
[17] munsell_0.5.0             n1qn1_6.0.1-2            
[19] gtable_0.2.0              R.methodsS3_1.7.1        
[21] devtools_1.13.6           mvnfast_0.2.5            
[23] memoise_1.1.0             labeling_0.3             
[25] inline_0.3.15             knitr_1.20               
[27] RcppArmadillo_0.8.600.0.0 rex_1.1.2                
[29] curl_3.2                  parallel_3.5.1           
[31] Rcpp_0.12.17              scales_0.5.0             
[33] jsonlite_1.5              SnakeCharmR_1.0.7        
[35] brew_1.0-6                ggforce_0.1.3            
[37] digest_0.6.15             stringi_1.1.7            
[39] dplyr_0.7.6               lbfgs_1.2.1              
[41] grid_3.5.1                tools_3.5.1              
[43] magrittr_1.5              dparser_0.1.8            
[45] lazyeval_0.2.1            tibble_1.4.2             
[47] PreciseSums_0.3           tidyr_0.8.1              
[49] pkgconfig_2.0.1           MASS_7.3-50              
[51] Matrix_1.2-14             assertthat_0.2.0         
[53] httr_1.3.1                R6_2.2.2                 
[55] units_0.6-0               nlme_3.1-137             
[57] git2r_0.22.1              compiler_3.5.1           
> 
```

</details>

To me the problem is with the following code bit:

```R
keyword <- string %>% 
    stringr::str_extract_all('@[[:alnum:]]+') %>% 
    purrr::flatten_chr() %>% 
    stringr::str_replace(stringr::fixed('@'), '') %>% 
    subset(!. %in% ignore_key)
```

**Comment by mattfidler** [comment]
I checked and that code isn't to blame, rather I can't get the aestheics from ggplot in the new version of ggplot.

**Comment by mattfidler** [comment]
https://github.com/tidyverse/ggplot2/blob/master/NEWS.md#breaking-changes

**Comment by guiastrennec** [comment]
Hi @mattfidler, sorry for the lack of response on my side I have been pretty busy lately. I actually fixed that issue in the latest (0.4.3) version of xpose on CRAN. Unfortunately I did not have time to update github but I will try to do it soon. Please let me know if that works.

**Comment by mattfidler** [comment]
![image](https://user-images.githubusercontent.com/514778/43986978-868d6f02-9cdf-11e8-940a-a21bb9084267.png)
 

Yes it works.  Hopefully you we get the github version updated soon.

**Comment by guiastrennec** [comment]
fixed in #107

**Comment by mattfidler** [comment]
Great!
